frontend/src/pages/AccountForm.tsx

```diff
@@ -12,15 +12,16 @@ import TextField from '@material-ui/core/TextField';
 
 import { User } from '../type-interfaces/User';
 import { getUsers, updateUser } from '../redux/actions/User';
+import {loadUser} from "../redux/actions/auth";
 
 interface Props {
   currentUser: User,
-  getUsers: Function,
+  loadUser: Function,
   updateUser: Function,
 }
 
 function AccountForm(props: any) {
-  const { currentUser, getUsers, updateUser } : Props = props;
+  const { currentUser, loadUser, updateUser } : Props = props;
   const [name, setName] = useState(currentUser.name);
   const [email, setEmail] = useState(currentUser.email);
   const [major, setMajor] = useState(currentUser.major);
```

Comment:
major, when clicked into after account creation (where its initial value is undefined or empty string), gives a big red error.

"Warning: A component is changing an uncontrolled input of type text to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components"